README.md

```diff
@@ -19,6 +19,11 @@
   <a href="https://slsa.dev"><img alt="SLSA 3" src="https://slsa.dev/images/gh-badge-level3.svg" /></a>
 </p>
 
+>[!Note]
+>**Zama 5-Question Developer Survey**
+>
+>We want to hear from you! Take 1 minute to share your thoughts and helping us enhance our documentation and libraries. 👉 **Click [here](https://www.zama.ai/developer-survey)** to participate. 
```

Comment:
```md
[Click here] 
```
to be consistant. Also the link is not always the same.

Reply:
Thanks!